docs/install-pmm/install-pmm-server/aws/run.md

```diff
@@ -1,59 +1,81 @@
+### Configure PMM Server to use an Elastic IP (Optional)
 
-You are creating a username and password that will be used for two purposes:
+For a static, public-facing IP address:
 {.power-number}
 
-1. authentication as a user to PMM - the credentials to log in to PMM.
+1. Allocate an Elastic IP address in the EC2 console.
+2. Associate the Elastic IP address with your EC2 instance's Network interface ID.
 
-2. authentication between PMM Server and PMM Clients - you will re-use these credentials on another host when configuring PMM Client for the first time on a server, for example (DO NOT RUN ON THIS PMM SERVER YOU JUST CREATED):
+!!! note alert alert-primary ""
+    Associating a new Elastic IP to an instance with an existing Elastic IP will disassociate the old one, but it will remain allocated to your account.
 
-    ```sh
-    pmm-admin config --server-insecure-tls --server-url=https://admin:admin@<IP Address>:443
-    ```
+For detailed information on EC2 instance IP addressing, see the [AWS documentation on using instance addressing](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-instance-addressing.html).
 
-    !!! note alert alert-primary ""
-        For instructions about how to access your instances by using an SSH client, see [Connecting to Your Linux Instance Using SSH](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html)
+## Resize the EBS volume
 
-    Make sure to replace the user name `ec2-user` used in this document with `admin`.
+To increase available disk space:
+{.power-number}
 
-## Resize the EBS Volume
+1. Increase the size of the EBS volume as needed. For instructions, see [Modifying the Size, IOPS, or Type of an EBS Volume on Linux](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-modify-volume.html).
+2. After updating the EBS volume, PMM Server will auto-detect changes within approximately 5 minutes and reconfigure itself.
 
-Your AWS instance comes with a predefined size which can become a limitation. To make more disk space available to your instance, you need to increase the size of the EBS volume as needed and then your instance will reconfigure itself to use the new size.
+## PMM Client configuration example
 
-The procedure of resizing EBS volumes is described in the Amazon documentation: [Modifying the Size, IOPS, or Type of an EBS Volume on Linux](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-modify-volume.html).
+When setting up PMM Client on another host, use a command similar to this (DO NOT RUN ON THE PMM SERVER):
 
-After the EBS volume is updated, PMM Server instance will auto-detect changes in approximately 5 minutes or less and will reconfigure itself for the updated conditions.
\ No newline at end of file
+```sh
+pmm-admin config --server-insecure-tls --server-url=https://admin:admin@<IP Address>:443
+```
+Replace `admin:admin` with your actual credentials and `<IP Address>` with your PMM Server's IP.
```

Comment:
do we really need to mention it? We have a whole separate block for it, don't we?